recipes/fmilib/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "2.0.2" %}
+
+package:
+  name: fmilib
+  version: {{ version }}
+
+source:
+  fn: FMILibrary-{{ version }}-src.zip
+  url: http://www.jmodelica.org/downloads/FMIL/FMILibrary-{{ version }}-src.zip
+  sha256: 85ff3358833818103d05e6d45f3fffcd2bbced9ef5f2262acac4d2de6523356e
+  patches:
+    - vc14-stdlib-conflict.patch  # [win and py35]
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - python  # [win]
+
+test:
+  requires:
+    - python {{PY_VER}}*  # [win]
+  commands:
+    - test -f $PREFIX/include/fmilib.h                                # [unix]
+    - test -f $PREFIX/lib/libfmilib.a                                 # [unix]
+    - if not exist %PREFIX%\\Library\\lib\\fmilib.lib exit 1          # [win]
+    - if not exist %PREFIX%\\Library\\include\\fmilib.h exit 1        # [win]
+
+about:
+  home: http://www.jmodelica.org/FMILibrary
+  license: BSD
```

Comment:
License version? Also can we package the license? Finally would suggest adding `license_family: BSD`.